packaging/installer/README.md

```diff
@@ -128,7 +111,7 @@ option when you install or update Netdata using the [automatic one-line installa
 script](#automatic-one-line-installation-script).
 
 ```bash
-bash <(curl -Ss https://my-netdata.io/kickstart.sh) --no-updates
+rm -f ./kickstart.sh ; wget https://my-netdata.io/kickstart.sh && sh ./kickstart.sh --no-updates
```

Comment:
Three no need in 'rm':

wget -O kickstart.sh https://my-netdata.io/kickstart.sh && sh ./kickstart.sh --no-updates

Reply:
The command needs to work even if they have an existing copy of the script present in the directory they are running it in, and we need it to guarantee that we get the newest copy of the script, so we need to remove any existing copy that may be present.

Reply:
I think this is addressed by `- O`.

Reply:
For the GNU implementation it is. For the busybox implementation I’m not 100% certain that it is.

Reply:
OK, confirmed now that it is the case for busybox as well.